plugins/gc/scheduler.go

```diff
@@ -331,6 +331,13 @@ func (s *gcScheduler) run(ctx context.Context) {
 			// runtime in between gc to reach the pause threshold.
 			// Pause threshold is always 0.0 < threshold <= 0.5
 			avg := float64(gcTimeSum) / float64(collections)
+			// anyway, on startup gcTimeSum might work fast and return 0
+			// on this case the algorithm turns in infinity loop which
+			// simple consume CPU on timer which fires without any delay.
+			// Use 5ms as fallback value to have interval 245ms for that case.
+			if avg == 0 {
```

Comment:
The `time.Duration` is actually in nanoseconds. It should never actually be zero here. Rather just comparing `< 5000000` it probably makes more sent to define a const like `const minimumGCTime = float64(5 * time.Millisecond)` and use that to compare less than and setting `avg`.